.github/ISSUE_TEMPLATE/epic.yaml

```diff
@@ -0,0 +1,46 @@
+name: ☔ Epic
+description: Use this template to propose a new feature or enhancement that requires multiple changes across the project.
+labels: [ epic ]
+title: ☔
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: textarea
+    attributes:
+      label: Problem Statement
+      description: Without specifying a solution, describe what the project is missing today.
+      placeholder: |
+        The rotating project logo has a fixed size and color.
+        There is no way to make it larger and more shiny.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Supporting documentation
+      description: |
+        Link to supporting documentation, such as design documents, user stories, or other relevant information.
+      placeholder: |
+        User stories:
+        - As a user, I want to control the size of the rotating logo using a CLI flag.
+        - As a user, I want to control the color of the rotating logo using a CLI flag.
+        [Design document](https://example.com/design)
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Acceptance Criteria
```

Comment:
I think we may want to add a `Testing` section here, which could be maybe integration, e2e, or manual for  the feature validation.

Reply:
This is something I considered. However, Looking at historical umbrella issues:

- https://github.com/Kong/kubernetes-ingress-controller/issues/6488
- https://github.com/Kong/kubernetes-ingress-controller/issues/6374
- https://github.com/Kong/kubernetes-ingress-controller/issues/5801

Testing is something that was included individually as past of each sub-task.

That being said I don't mind adding it if we think it's worth it.

Reply:
Yes. I think Epic is a high-level content, and adding tests to the split subtasks may be more specific.